doc/sphinx-guides/source/installation/config.rst

```diff
@@ -1027,8 +1027,8 @@ dataverse.siteUrl
 	``<jvm-options>-Ddataverse.fqdn=dataverse.example.edu</jvm-options>``
 	``<jvm-options>-Ddataverse.siteUrl=http://${dataverse.fqdn}:8080</jvm-options>``
 
-dataverse.files.directory
-+++++++++++++++++++++++++
+dataverse.files.id.directory
+++++++++++++++++++++++++++++
 
 This is how you configure the path to which files uploaded by users are stored.
```

Comment:
I might leave the dataverse.files.directory as shown (remove the id) and change this explanatory text to:
This is how you configure the path Dataverse uses for temporary files. (File store specific dataverse.files.\<id\>.directory options set the permanent data storage locations.)